bookmark.html

```diff
@@ -4,8 +4,69 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <a href="./index.html">index</a>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Kdam+Thmor+Pro&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
     <title>Document</title>
   </head>
-  <body></body>
+  <main>
+    <body>
+      <h1>BOOKMARKS</h1>
+      <section>
+        <article class="question-section">
+          <img
+            class="box-img"
+            src="images/bookmark_white_24dp(1).svg"
+            alt="bookmark-icon"
+          />
+          <h2>Question</h2>
+          <p>This is where questions being saved!</p>
+        </article>
+      </section>
+      <br />
```

Comment:
We don't use `br` elements between elements. The only practice where it is ok to work with is for example to make a break in the content of your `p` element. [See MDN for further information.](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br)